lambda/src/main/scala/pricemigrationengine/handlers/MigrationHandler.scala

```diff
@@ -12,11 +12,10 @@ object MigrationHandler extends ZIOAppDefault with RequestHandler[Unit, Unit] {
 
   private val migrateActiveCohorts =
     (for {
-      today <- Clock.currentDateTime.map(_.toLocalDate)
       cohortSpecs <- CohortSpecTable.fetchAll
       activeSpecs <-
         ZIO
-          .filter(cohortSpecs)(cohort => ZIO.succeed(CohortSpec.isActive(cohort)(today)))
+          .filter(cohortSpecs)(cohort => ZIO.succeed(true)) // TODO: simplify this
```

Comment:
do we need this line still?

Reply:
@tomrf1 
Technically no, but when I removed it the result didn't compile so I decided to refactor the entire block in another step.